Automattic/jetpack
# My Jetpack: Update AI interstitial background

Closes #30979 

## Proposed changes:

* Add support for `imageContainerClassName`.
* Replace `jetpack-ai.png` image.
* Add default background for `AI Interstitial`.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






* Goes to `My Jetpack`
* Navigate to AI Interstitial (`#/add-jetpack-ai` route).
* You should see the new background.
